pkgs/development/python-modules/quart/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, click
+, aiofiles
+, werkzeug
+, jinja2
+, hypercorn
+, itsdangerous
+, blinker
+, mock
+, hypothesis
+, pytest
+, pytest-asyncio
+, pytest-cov
+, pytest-sugar
+}:
+
+buildPythonPackage rec {
+  pname = "quart";
+  version = "0.15.1";
```

```suggestion
  version = "0.15.1";
  disabled = pythonOlder "3.7";
```

see https://gitlab.com/pgjones/quart/-/blob/main/setup.py#L7